src/public/js/components/sideBar/component/Code/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import classNames from 'classnames';
-import SyntaxHighlighter from 'react-syntax-highlighter';
-import { atomOneLight } from 'react-syntax-highlighter/styles/hljs';
+import SyntaxHighlighter from 'react-syntax-highlighter/prism';
+import { coy } from 'react-syntax-highlighter/styles/prism';
```

Comment:
@idoo I am getting 
```
ERROR in ./js/components/sideBar/component/Code/index.js
Module not found: Error: Can't resolve 'react-syntax-highlighter/styles/prism' in '/Users/bliashenko/Learning/codecrumbs/src/public/js/components/sideBar/component/Code'
 @ ./js/components/sideBar/component/Code/index.js 9:0-60 79:15-18
 @ ./js/components/sideBar/component/SideBar.js
 @ ./js/components/sideBar/SideBarContainer.js
 @ ./js/App.js
 @ ./js/index.js

```
^^ `yarn`\`yarn --no-lockfile`didn't help

Reply:
@Bogdan-Lyashenko try yarn upgrade?

could you check version please? it should be 11.x